config/form_profile_mappings/5655.yml

```diff
@@ -1,3 +1,6 @@
+personalIdentification:
+  sSN: [identity_information, ssn_last_four]
```

Comment:
Should this be all caps? or camelCased? This feels weird. 😕

Reply:
It is kind of weird, but if you are referring to the keys being camelCased and values within the array represented as snake_case that is an established pattern within other form profile mappings.

Reply:
Oh. So it's coming in as `s_s_n`? 😲

Reply:
so when the frontend hits the in_progress_form endpoint (in this case it would be a GET /v0/in_progress_forms/5655) the keys here will be rendered to the frontend in camelCase as they are intended to.

These could be changed, but I'd rather stick as close as possible to what the DMC is expecting once we get to the POST.

The values within the arrays reference Virtus models and associated methods (i.e. FormIdentityInformation class and the ssn_last_four method) that method gets called to populate the `sSN` key which is what the DMC is expecting.

I could represent this as `ssn: [identity_information, ssn_last_four]` but then on the POST either the frontend would need to alter the key to sSN or the backend would prior to POSTing to the DMC.

I would like to get them to change it, but for right now it should stay as it is in my opinion.